ecl/hqlcpp/hqlhtcpp.ipp

```diff
@@ -58,6 +58,48 @@ private:
     bool grouped;
 };
 
+class GlobalCompareClass
```

Comment:
So the GlobalClassBuilder doesn't really have a class that represents the class being created it is one that knows how to build a class like that.